.github/workflows/covector-version-or-publish.yml

```diff
@@ -10,6 +10,7 @@ on:
   push:
     branches:
       - dev
+      - dev-refactor
```

Comment:
Why do we need the covector workflow for dev-refactor?

Reply:
Agreed. That shouldn't be part of `dev-refactor`.